fdbcli/DebugCommands.actor.cpp

```diff
@@ -424,9 +425,9 @@ ACTOR Future<bool> doCheckAll(Database cx, KeyRange inputRange, bool checkAll) {
 			break;
 
 		} catch (Error& e) {
-			printf("Error: %s", e.what());
+			fmt::print("Error: %s", e.what());
```

Comment:
Does `fmt::print` work with `%s`? I have only seen it with `{}`. I can also check later.

Reply:
I do not think so, I missed it, will fix.